x-pack/plugins/lens/public/xy_visualization/xy_config_panel.tsx

```diff
@@ -68,6 +86,74 @@ export function LayerContextMenu(props: VisualizationLayerWidgetProps<State>) {
   );
 }
 
+export function XyToolbar(props: VisualizationToolbarProps<State>) {
+  const [open, setOpen] = useState(false);
+  const hasNonBarSeries = props.state?.layers.some(
+    (layer) => layer.seriesType === 'line' || layer.seriesType === 'area'
+  );
+  return (
+    <EuiFlexGroup justifyContent="flexEnd">
+      <EuiFlexItem grow={false}>
+        <EuiPopover
+          panelClassName="lnsXyToolbar__popover"
+          button={
+            <EuiButtonEmpty
+              iconType="arrowDown"
+              iconSide="right"
+              onClick={() => {
+                setOpen(!open);
+              }}
+            >
+              <EuiIcon type="gear" />
```

Comment:
Let's just be explicit here. Also can you make the button similar to how you did the chart select?

```suggestion
              Settings
```